CHANGELOG.md

```diff
@@ -10,7 +10,7 @@
 
 ### Fixed
 - Bug in how integration_time array is set in setup
-
+- Circular-installation bug
```

Comment:
I think the convention is to put more recent changes higher on the list.

Reply:
Ah damnit, got that wrong last time as well. Fixed in be478c0a4e33a6d80fc014aa6a1d256d1dd4e229